javascript-light-bulb/main.js

```diff
@@ -0,0 +1,18 @@
+
+var $body = document.querySelector('.light');
+var $container = document.querySelector('.switch-container-light');
+var $switch = document.querySelector('.switch-light');
+var count = 0;
+
+$switch.addEventListener('click', function (event) {
+  count++;
+  if (count % 2 === 1) {
```

Comment:
Instead of doing this with a counter, you should try doing this with a boolean. A good name for the variable would be `isOff` or `isOn`, with a boolean value matching the current state of the light bulb.